cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-12-19 15:13
+
+import cinema.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(blank=True, null=True, upload_to=cinema.models.convert_uploded_image_name),

```

Comment:
Ensure that the `convert_uploded_image_name` function exists in `cinema.models` and is correctly implemented. This function is crucial for handling the image upload path and naming.